qiskit/backends/local/localprovider.py

```diff
@@ -63,6 +67,27 @@ def available_backends(self, filters=None):
                         if instance.configuration.get(key) == value}
         return list(backends.values())
 
+    def aliased_backend_names(self):
+        return {
+            'local_qasm_simulator': ['local_qasm_simulator_cpp',
+                                     'local_qasm_simulator_projectq',
+                                     'local_qasm_simulator_py'],
+            'local_statevector_simulator': ['local_statevector_simulator_cpp',
+                                            'local_statevector_simulator_projectq',
+                                            'local_statevector_simulator_py',
+                                            'local_statevector_simulator_sympy'],
+            'local_unitary_simulator': ['local_unitary_simulator_cpp',
+                                        'local_unitary_simulator_py',
+                                        'local_unitary_simulator_sympy'],
+            'local_clifford_simulator': ['local_clifford_simulator_cpp']
```

Comment:
I don't think the sympy and projectq simulators be aliased here as "standard" sdk ones, they should probably be loaded loaded separately as external simulators since they either have different functionality, or are not maintained by us.

Reply:
Right now they are part of localprovider. But we will eventually make them 3rd party providers. Until then, I think this should stay to make sure what is in localprovider works properly. Also as a guide for whoever ports it to the 3rd party providers.